internal/server/property_value.go

```diff
@@ -125,7 +126,10 @@ func getPropertyValuesHelper(
 			return propVals.Nodes, nil
 		}, nil)
 	for dcid := range branchNodeMap {
-		branchNodes := branchNodeMap[dcid].([]*Node)
+		branchNodes, ok := branchNodeMap[dcid].([]*Node)
+		if !ok {
+			return nil, fmt.Errorf("branch nodes invalid for %s", dcid)
```

Comment:
For my understanding, why do we return error in this case, while silently putting in nil in others?